Yandex Analytics Adapter: initial release (#5052)
prebid/prebid.github.io: dev-docs/analytics/yandex.md

---
layout: analytics
title: Yandex Metrica

> description: Track your Prebid.js in Yandex Metrica - Top-5 worldwide web analytics tool.

Reviewer: ```suggestion
description: Track your Prebid.js in Yandex Metrica
```

prebid.org is a technical documentation, hence we restrict marketing claims

> modulecode: generic

Reviewer: ```suggestion
modulecode: yandexAnalytics
```

As your adapter is configured via `yandexAnalytics`, it's not the generic one mentioned here: https://docs.prebid.org/dev-docs/modules/genericAnalyticsAdapter.html right?

---

# Description

> This adapter is designed to work with [Yandex Metrica](https://metrica.yandex.com/about) - Top-5 worldwide web analytics tool.

Reviewer: ```suggestion
This adapter is designed to work with [Yandex Metrica](https://metrica.yandex.com/about) - a web analytics tool.
```

> Disclosure: provider loads Metrica Tag build based on [github.com/yandex/metrica-tag](https://github.com/yandex/metrica-tag), ~60 kB gzipped.

Reviewer: ```suggestion
{: .alert.alert-warning :}
Disclosure: provider loads Metrica Tag build based on [github.com/yandex/metrica-tag](https://github.com/yandex/metrica-tag), ~60 kB gzipped.
```
